### ISSUE TITLE
Improve documenation to locate/modify deployed collector's configuration

### ISSUE THREAD

**Issue by rogercoll** [elastic/opentelemetry-demo#84]

**Comment by krol3** [comment]
Happy to help with this issue. It will be using openTelemetry operator installed by helm, right?

**Comment by rogercoll** [comment]
> Happy to help with this issue. It will be using openTelemetry operator installed by helm, right?

I don't think so, the upstream and the Elastics fork uses the [open-telemetry/opentelemetry-demo](https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-demo) Helm Chart, which uses the  [open-telemetry/opentelemetry-collector](https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-collector) Chart to deploy the collector. The last, does not use the operator but raw daemonset and/or deployments.
